### PR TITLE
kernel: Always build the kernel version found locally for CI

### PR DESCRIPTION
**Pull request by ganeshmaharaj**
Fixes: #1441

Signed-off-by: Ganesh Maharaj Mahalingam <ganesh.mahalingam@intel.com>

### CODE REVIEW

**Comment by ganeshmaharaj** [comment]
@jcvenegas the patch to address https://github.com/kata-containers/packaging/issues/431

File: .ci/install_kata_kernel.sh [hunk 1]
```diff
@@ -65,9 +65,10 @@ get_packaged_kernel_version() {
 }
 
 build_and_install_kernel() {
+	# Always build and install the kernel version found locally
 	info "Install kernel from sources"
 	pushd "${tmp_dir}" >> /dev/null
-	"${kernel_repo_dir}/kernel/build-kernel.sh" "setup"
+	"${kernel_repo_dir}/kernel/build-kernel.sh" "setup" -v "${kernel_version}"
 	"${kernel_repo_dir}/kernel/build-kernel.sh" "build"
```
Comment (jcvenegas) [inline .ci/install_kata_kernel.sh]: I'd add the same to build and install as needs to find the kernel path based on the kernel version.

File: .ci/install_kata_kernel.sh [hunk 2]
```diff
@@ -65,9 +65,10 @@ get_packaged_kernel_version() {
 }
 
 build_and_install_kernel() {
+	# Always build and install the kernel version found locally
 	info "Install kernel from sources"
 	pushd "${tmp_dir}" >> /dev/null
-	"${kernel_repo_dir}/kernel/build-kernel.sh" "setup"
+	"${kernel_repo_dir}/kernel/build-kernel.sh" "setup" -v "${kernel_version}"
```
Comment (jcvenegas) [inline .ci/install_kata_kernel.sh]: Also I am not sure how well bash will handle this,  I prefer set this as:
` ./build-kernel.sh -v "${kernel_version} setup`

**Comment by chavafg** [comment]
/retest

seems that we lost some triggered jobs... relaunching.

**Pull request merged**